### DISCUSSION TITLE
How to persist dom updates on pdf.js redraws?

### DISCUSSION THREAD

**Discussion by chan-dev** [stephanrauh/ngx-extended-pdf-viewer#960] [Q&A]
We're currently highlighting some selected text and wrapping them in custom markup but the modifications are removed on some sort of like redraw events happening (e.g rotate, zoom, etc).

Is there a way to keep modifications during such events?

**Comment by stephanrauh** [comment]
I don't think so. But you can register the `(pageRendered)` or `(textLayerRendered)` event and re-apply your modifications in the event handler.
